default/python/AI/PriorityAI.py

```diff
@@ -244,9 +244,14 @@ def _calculate_colonisation_priority():
     colonizers = list(ColonisationAI.empire_colonizers)
     if "SP_SLY" not in colonizers and outpost_prio > 0:
         return 0.0
-    colony_opportunities = [species_name for (_, (score, species_name)) in foAI.foAIstate.colonisablePlanetIDs.items()
-                            if score > 60]
-    num_colonisable_planet_ids = len(colony_opportunities)
+    min_score = ColonisationAI.MINIMUM_COLONY_SCORE
+    minimal_range = 2  # one more than the conditional floor set by ColonisationAI.revise_threat_factor()
+    minimal_opportunities = [species_name for (_, (score, species_name)) in foAI.foAIstate.colonisablePlanetIDs.items()
+                             if min_score < score <= min_score + minimal_range]
```

Comment:
Pretty sure that's a typo.

Reply:
Could you be more specific?  The typo is not leaping out at me...